src/Console/Commands/DefaultCommand.php

```diff
@@ -105,4 +124,26 @@ private function renderIssue(OutputInterface $output, Issue $issue, string $curr
             </div>
         HTML);
     }
+
+    /**
+     * Extract a string from the array of suggestions
+     */
+    private function extractSuggestionsString(Issue $issue): string
+    {
+        $suggestions = $issue->misspelling->suggestions;
+        $suggestionCount = count($suggestions);
+
+        // I tried using a match statement here, but it didn't work as expected
+        if ($suggestionCount > 1) {
```

Comment:
This looks good! I just had a thought if it could be shortened using a match statement and came up with this:

```php
$suggestions = $issue->misspelling->suggestions;

return match (true) {
    count($suggestions) >= 1  => sprintf('%s or %s?', implode(', ', array_slice($suggestions, 0, -1)), end($suggestions)),
    count($suggestions) === 1 => sprintf('%s?', $suggestions[0]),
    count($suggestions) === 0 => 'Wow! Sorry - but there are no suggestions for this misspelling.',
};
```

This isn't necessarly more clear or better, but it has less variables that are only used in one spot.